development.ini

```diff
@@ -11,7 +11,7 @@ pyramid.default_locale_name = en
 
 jujugui.sandbox = true
 jujugui.raw = true
-jujugui.combine = true
+jujugui.combine = false
```

Comment:
JC had a good point about not committing this - we usually run into issues with the 'combine' assets but don't know about it until we try to release to production.